.github/workflows/ci.yml

```diff
@@ -224,6 +237,24 @@ jobs:
         echo "CPPINTEROP_DIR=$CPPINTEROP_DIR" >> $GITHUB_ENV
         echo "LLVM_BUILD_DIR=$LLVM_BUILD_DIR" >> $GITHUB_ENV
         echo "CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH" >> $GITHUB_ENV
+
+    - name: Run clang-tidy
+      if: ${{ (matrix.clang-tidy == true) }}
+      uses: ZedThree/clang-tidy-review@v0.13.2
```

Comment:
Hi @Smit1603, this action runs the `cmake` command in a docker container, so any dependencies that you install in the host machine won't be available to the container while building. 
So, to fix this, you should install these dependencies inside the clang-tidy step itself, you can use the `apt_packages` option to install these dependencies ([reference](https://github.com/ZedThree/clang-tidy-review)).

Reply:
Ohhh yeah, I understood, actually I was plaining to create a seperate docker container to resolve but this might be best I feel. Will try this